calibration:
  base_frame: "base/base_link"
  shooting_box_frame: "shooting_box_frame"
  target_box_frame: "target_box_frame"
  shooting_box_center_red: [1325, 1125, 0]
